tests/utils/esx/vdvs.go

```diff
@@ -15,7 +15,7 @@
 // This util provides various helper methods that can be used by different tests to
 // fetch information related to vSphere docker-volume-service.
 
-package verification
+package esx
```

Comment:
@govint  It was placed earlier at `utils/verification` package ... is it a good idea to move back to `utils/verification` package?